core/src/main/scala/precog/SbtPrecogBase.scala

```diff
@@ -412,7 +412,7 @@ abstract class SbtPrecogBase extends AutoPlugin {
             parsed foreach {
               case (key, value) =>
                 println(s"::add-mask::$value")
-                println(s"::$key=$value::>>::" + "$GITHUB_ENV")
+                println(s"""::"$key=$value"::>>::$$GITHUB_ENV""")
```

Comment:
But shouldn't there be an `echo`?

Reply:
`set-env` also required an `echo`; so I was operating under the assumption that the `echo` is being brought in from elsewhere

Reply:
Oh wait, I think you're right; I was looking at `set-env` in a different context, the context where we use a `WorkflowStep`